projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -26,7 +26,16 @@ function register_block() {
 	) {
 		Blocks::jetpack_register_block(
 			BLOCK_NAME,
-			array( 'render_callback' => __NAMESPACE__ . '\render_block' )
+			array(
+				'render_callback' => __NAMESPACE__ . '\render_block',
+				'supports'        => array(
```

Comment:
It would be great to refactor to `block.json` so you don't have to copy the same data between JavaScript and PHP.

Reply:
Indeed. However, we would need to refactor all of Jetpack's block registration and enqueuing process to make that happen. That will most likely need to happen later this year as we extract some of our blocks out of the bundle to ship them separately. For now, I think this is an acceptable solution here (and for changes to our other blocks).

Reply:
We at @Automattic/apex would be interested to jump on the project of refactoring work as it's been in our discussions when we were looking at the state of all of the Jetpack blocks.

Reply:
`block.json` was introduced as of WordPress 5.5. As of WordPress 5.8, it's the recommended way to register blocks with `register_block_type` on the server. More details in https://make.wordpress.org/core/2021/06/23/block-api-enhancements-in-wordpress-5-8/.

For Global Styles, it's better when all blocks are registered on the server because block supports is used to generate CSS styles on the fly.

Reply:
Thanks for the docs link and references!